### PR TITLE
Megabar

### LINKED ISSUE #363: [FEATURE] Megamenu component
A "Megamenu" is a special dropdown menu that allows to display a large number of menu items, as shown on the [Bootstrap Italia documentation page for the Megamenu component](https://italia.github.io/bootstrap-italia/docs/menu-di-navigazione/megamenu/).

We'd like to create a Megamenu component for the React kit, that replicates* Bootstrap Italia behaviour.

_* Completing the mobile version is part of this issue, too._

### PR DESCRIPTION
**Pull request by Pratyush1197**
Fixes #363

### CODE REVIEW

File: assets/css/fonts.css [hunk 1]
```diff
@@ -179,3 +179,23 @@
     url(../../node_modules/bootstrap-italia/dist/fonts/Roboto_Mono/RobotoMono-Bold.ttf) format('truetype');
     unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
+.megamenu{
```
Comment (dej611) [inline assets/css/fonts.css]: Remove all this CSS. The only CSS allowed is the BI package.

Comment (Pratyush1197) [reply assets/css/fonts.css]: So what should I use instead of CSS? If I have to set the width of container?

**Comment by dej611** [comment]
@Pratyush1197 are you still working at this?

**Comment by Pratyush1197** [comment]
Yes sir due to my semester examinations I am unable to do it now. I will continue it as soon as possible

**Comment by dej611** [comment]
Any update on this?

**Pull request closed without merging**